src/AppPrintOnly.test.tsx

```diff
@@ -38,6 +38,8 @@ beforeEach(() => {
 
 jest.useFakeTimers()
 
+jest.setTimeout(10000)
```

Comment:
I'm a little worried about increasing timeout without being certain that we know why.

Reply:
it's just more printing :) That is all.